projects/sunbird-epub-player/src/lib/epub-viewer/epub-viewer.component.html

```diff
@@ -0,0 +1 @@
+<div id="content" #epubViewer></div>
```

Comment:
when i say content id means the **content identifier** like **do_123456** as id set